src/bun.js/webcore/streams.zig

```diff
@@ -2760,10 +2760,10 @@ pub fn HTTPServerWritable(comptime ssl: bool) type {
 
         pub fn onAborted(this: *@This(), _: *UWSResponse) void {
             log("onAborted()", .{});
-            this.done = true;
```

Comment:
why?

Reply:
to keep the same behavior as endFromJS

Reply:
reverted to original behavior in https://github.com/oven-sh/bun/pull/2261/commits/95ffe1a18bb01dfe5693ef494d3ca470cdfcac93